configs/lisk/mainnet.yml

```diff
@@ -0,0 +1,243 @@
+  explorerHostname: api.etherscan.io
+  explorerTokenEnv: ETHERSCAN_TOKEN
+  contracts:
+    l1TokenBridge:
+      name: L1ERC20TokenBridge
+      address: *l1TokenBridge
+      proxyName: OssifiableProxy
+      implementation: *l1TokenBridgeImpl
+      proxyChecks:
+        proxy__getAdmin: *l1Agent
+        proxy__getImplementation: *l1TokenBridgeImpl
+        proxy__getIsOssified: false
+      ozNonEnumerableAcl:
+        *DEFAULT_ADMIN_ROLE : [ *l1Agent ]
+        *DEPOSITS_DISABLER_ROLE : [ *l1Agent, *l1EmergencyBreaksMultisig ]
+        *DEPOSITS_ENABLER_ROLE : [ *l1Agent ]
+        *WITHDRAWALS_DISABLER_ROLE : [ *l1Agent, *l1EmergencyBreaksMultisig ]
+        *WITHDRAWALS_ENABLER_ROLE : [ *l1Agent ]
+      checks:
+        DEFAULT_ADMIN_ROLE: *DEFAULT_ADMIN_ROLE
+        DEPOSITS_DISABLER_ROLE: *DEPOSITS_DISABLER_ROLE
+        DEPOSITS_ENABLER_ROLE: *DEPOSITS_ENABLER_ROLE
+        WITHDRAWALS_DISABLER_ROLE: *WITHDRAWALS_DISABLER_ROLE
+        WITHDRAWALS_ENABLER_ROLE: *WITHDRAWALS_ENABLER_ROLE
+        getRoleAdmin:
+          - args: [ *DEFAULT_ADMIN_ROLE ]
+            result: *DEFAULT_ADMIN_ROLE
+          - args: [ *DEPOSITS_DISABLER_ROLE ]
+            result: *DEFAULT_ADMIN_ROLE
+          - args: [ *DEPOSITS_ENABLER_ROLE ]
+            result: *DEFAULT_ADMIN_ROLE
+          - args: [ *WITHDRAWALS_DISABLER_ROLE ]
+            result: *DEFAULT_ADMIN_ROLE
+          - args: [ *WITHDRAWALS_ENABLER_ROLE ]
+            result: *DEFAULT_ADMIN_ROLE
+        hasRole: null # checked automatically by ozNonEnumerableAcl section
+        isDepositsEnabled: true
+        isInitialized: true
+        isWithdrawalsEnabled: true
+        l1Token: *l1Wsteth
+        l2Token: *l2Wsteth
+        l2TokenBridge: *l2TokenBridge
+        messenger: *l1CrossDomainMessenger
+        supportsInterface: null
+      implementationChecks:
+        isInitialized: false
+        hasRole:
+          args: [*DEFAULT_ADMIN_ROLE, *ZERO_ADDRESS]
+          result: false
+        isDepositsEnabled: false
+        isWithdrawalsEnabled: false
+        l1Token: *l1Wsteth
+        l2Token: *l2Wsteth
+        l2TokenBridge: *l2TokenBridge
+
+    l1EmergencyBreaksMultisigContract:
+      name: GnosisSafe
+      proxyName: GnosisSafeProxy
+      address: *l1EmergencyBreaksMultisig
+      checks:
+        getOwners:
+          - *multisigOwner1
+          - *multisigOwner2
+          - *multisigOwner3
+          - *multisigOwner4
+          - *multisigOwner5
+        getThreshold: 3
+        VERSION: null
+        approvedHashes: null
+        checkNSignatures: null
+        checkSignatures: null
+        domainSeparator: null
+        encodeTransactionData: null
+        getChainId: null
+        getModulesPaginated: null
+        getStorageAt: null
+        getTransactionHash: null
+        isModuleEnabled: null
+        isOwner: null
+        nonce: null
+        signedMessages: null
+
+l2:
+  rpcUrl: https://rpc.api.lisk.com
+  explorerHostname: blockscout.lisk.com
```

Comment:
If L1 explorer host name is set to:
`api.etherscan.io`,

should then L2 explorer host name be set to:
`blockscout.lisk.com/api`?